11-if/questions/3-if-edit.md

```diff
@@ -0,0 +1,224 @@
+# Questions: `if` #
+
+## 1. What does "control flow" mean?
+(A) Changing the top-to-bottom execution of a program <br>
+(B) Changing the left-to-right execution of a program <br>
+(C) Deciding which statements are executed <br>
+
+---
+**Solution**: (C) <br>
+
+Go will always execute code from top-to-bottom going left-to-right on each line. Control flow allows us decide which parts of a program will run depending on conditional values like `true` and `false`.
+
+---
+
+## 2. How can the `if` statement below be simplified?
+```go
+if (mood == "perfect") {
+    // this code is not important
+}
+```
+1. `if {mood == "perfect"}`
+2. `if [mood == "perfect"]`
+3. `if mood = "perfect"`
+4. `if mood == "perfect"` *CORRECT*
+
+---
+
+**Solution**: (D) <br>
+Go does not required parentheses for conditional statements. <br>
+(B) Using `{}` and `[]` will is will throw a syntax error. <br> 
+(C) `=` is an assignment operator and cannot be used as a condition. <br>
+
+---
+
+## 3. Which of the following choices will allow the code to run? ##
+
+```go
+package main
+
+import "fmt"
+
+func main() {
+    // this program prints "cool"
+    // when the mood is "happy"
+
+    mood := "happy"
+
+    if "happy" {
+        fmt.Println("cool")
+    }
+}
+```
+(A) Wrapping the `"happy"`` after the `if` inside parentheses <br>
+(B) Comparing "happy" with the variable `mood`: `if mood == "happy"` <br>
+(C) Directly using `mood` instead of `"happy"`: `if mood {...}` <br>
+(D) Trick question - no changes are required for the program to run. <br>
+
+---
+
+**Solution**: (B) <br>
+By using `==`, the equality operator, we allow are program to continue executing because the logic within the `if` block. <br>
+
+---
+
+## 4. How can the following program be simplified? ##
+**Hint**: Only one condition needs to be changed.
+
+```go
+package main
+
+import "fmt"
+
+func main() {
+    happy := true
+
+    if happy == true {
+        fmt.Println("cool!")
+    }
+}
+```
+(A) `happy != false` <br>
+(B) `!happy == false` <br>
+(C) `happy` <br>
+(D) `!happy == true` <br>
+
+---
+
+**Solution**: (C) <br>
+The value of happy is already `true`, making the condition `true` as well, which will call the print function. <br>
+`!happy == true` returns false so the program will not print anything. <br>
+`happy != false` and `!happy == false` are also valid but are verbose compared to (C). <br>
+
+---
+
+## 5. How can the following program be simplified? ##
+```go
+package main
+
+import "fmt"
+
+func main() {
+    happy != false
+    
+    if happy == !true {
+        fmt.Println("why not?")
+    }
+}
+```
+(A) Changing `happy != false` to `happy == !true` <br>
+(B) `!happy` <br>
+(C) `happy == false` <br>
+(D) `!happy == false` <br>
+
+---
+
+**Solution**: (B) <br>
+`!happy` already returns`true` because the value is changed to `false` or `!true`. <br>
```

Comment:
You need to close the code formatting at the end of the false.

Perfect! `!happy` already. returns `true` because it is already `false`. There is no need to compare the value to `false` or `!true`.